<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8">
    <title>Eve Zeyl Fiskebeck Resume</title>
	<link href="css/styles.css" type="text/css" rel="stylesheet" />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
    <script src='https://kit.fontawesome.com/a076d05399.js'></script>
</head>

<!--visible elements structure-->
<body>
    <main>
        <div class="container">
            <section id="contact" class="side">
                <img src="files/image1.png" width="100%" class="center"/>
                <br>

                <div id="contact-icons" class="contact-icons">
                    <i class="fa fa-address-book"> Gjøstigen 18.<br>1914 Ytre Enebakk.</i>
                    <br>
                    <i class="fa fa-birthday-cake"> 27/07/76 </i>
                    <br>
                    <i class="fa fa-phone"> +47 454 15 690 </i>
                    <br>
                    <i class="fa fa-envelope"> <a href="mailto:evezeyl@gmail.com">evezeyl@gmail.com</a> </i>
                    <br>
                    <i class="fab fa-github"> <a href="https://github.com/evezeyl">Github</a></i>
                    <br>
                    <i class="fab fa-linkedin" style="color:#2E86C1"> <a href="https://no.linkedin.com/in/eve-fiskebeck-73a29bb/no">LinkedIn</a></i>
                    <br>
                    <i class='fab fa-researchgate' style="color:#A3E4D7"> <a href="https://www.researchgate.net/profile/Eve_Zeyl_Fiskebeck">Researchgate</a></i>
                </div>
                <br>

                <!-- finish
                <div class ="skills">
                    <h5>Programming skills</h5>
                    <img src="files/skill-bar.jpg" width="100%" class="center">
                </div>
                -->

            </section>


            </section>
            <section class="title">
                <h1 align="center">Eve Zeyl Fiskebeck</h1>
            </section>

            <section class ="nutshell">
                <h2>In a nutshell:</h2>
                <ul>
                    <li>I want to use molecular genetics analysis methods (phylogeny and phylogeography, transmission and GWAS) to study micro and macro evolution mechanisms in order to understand adaptation and adaptability potential of different organisms to provide a better understanding of their biology and in case of pathogens to improve detection and help informed decisions for control.</li>
                    <li>I am interested in finding the best solutions to analyse data, understanding methods, combining methods, transposing methods between different scientific fields.</li>
                    <li>My abilities are equally "practical" and "theoretical". I see and focus on methodologies which have practical application in the context of a current subject.</li>
                    <li>I particularly thrives in a coworking environment where each person provides its own expertise and contribute as a piece of a jigsaw puzzle to create a complete picture of a scientific question.</li>
                    <li>I am enthusiastic and likes to help others. I believe in exchange of competencies. I thrive with constant learning.</li>
                    <li>I enjoy coding and improving my data science abilities (relaxing self-fulfilling activities).</li>
                </ul>
            </section>

            <section class = "experience">
                <h2>Working experience:</h2>
                <section id="VI">
                    <p class="greyband">01/12/2018 to 30/11/2019 Reconversion to Bioinformatics. Trainee and 40% employment (researcher) during 6 months on 2 projects. Veterinary Institute, Oslo, Norway.</p>
                    <ul>
                        <li>Phylogenetics and phylogeography of <em>Yersinia ruckeri</em>.</li>
                        <li>Phylogenetics of <em>Listeria monocytogenes</em>.</li>
                    </ul>
                    <p>Tasks:</p>
                    <ul>
                        <li>Exploration methods: Assembly, mapping, annotation, variants/SNP typing, dendrograms and phylogenetic trees (core and pangenome), phylogeography and transmission methods. GWAS.</li>
                        <li>Scripting: R (mainly), bash, python</li>
                        <li>Contribution to intern course: Conda, Poppunk, HPC, mapping, tree building ...</li>
                    </ul>
                    <p>Reference:  <a href="mailto:karin.lagesen@vetinst.no">  Karin Lagesen  </a> <i class="fa fa-phone fa-sm"> +47 915 75 916</i></p>
                </section>
                <section id="No-engineer">
                    <p class="greyband">2009 to 2014 Laboratory Engineer. University of Oslo, Natural History Museum (NHM) and 2004 to 2010 Ph.D project.
                        <ul>
                            <li>Project 3: <a href="http://folk.uio.no/bachmann/artsprosjekt%20-%20Kinorhyncha.html">"Kinorhyncha - a poorly and neglected animal phylum"</a> - artsprosjektet - managed by Artsdatabanken.</li>
                            <li>Project 2: <a href="http://folk.uio.no/bachmann/artsprosjekt%20-%20Gyrodactylus.html">"Diversity mapping of Norwegian Gyrodactylid flatworms - mining natural history collections."</a></li>
                            <li>Prosjekt 1 and PhD. project: Analyse of new material & analyses polar bears for Ph.D. Total > 900 samples).</li>
                        </ul>
                    <p>Tasks:</p>
                        <ul>
                            <li>Sampling, microscope slides Preparations, microscopy, digitalisation.</li>
                            <li>Testing and improving methods for DNA isolation and analysis (skin, plasma, blood).</li>
                            <li>Routine DNA isolation, quantification, PCR, multiplex PCR, Sanger Sequencing, Genotyping, Sanger sequencing, sequence alignment, barcoding ...</li>
                            <li>Competence transfer:</li>
                            <ul>
                                <li>instructor: introduction to laboratory courses for Ph.D. and master students (2 weeks, 2 different years): RNA and DNA isolation, quantification, PCR, reverse transcription, sanger sequencing, sequence visualising (staden package).</li>
                                <li>Advise and help for master students (Population genetics analyses, Kinorhycha).</li>
                            </ul>
                            <li>Statistical and data analysis.</li>
                            <li>Contributions to publishing (all projects).</li>
                            <li>Quality control of a part of museum collection.</li>
                        </ul>
                </section>
                <section id="Fr-engineer">
                    <p class="greyband">2003-2004 Laboratory Engineer. France Laboratoire d’Ecologie Alpine, CNRS, Grenoble.</p>
                    <p>Tasks:</p>
                    <ul>
                        <li>Testing and developing new markers for wildlife forensics.</li>
                        <li>DNA isolation, sequencing, SNPs genotyping, species identification (barcoding).</li>
                        <li>Contribution to daily laboratory management.</li>
                        <li>Competence transfer: training of a bachelor student to laboratory work (1 month).</li>
                    </ul>
                </section>

            <section id= education class ="education">
                <h2>Education</h2>
                <section id="PhD">
                    <p class="greyband">End 2004 to 2010. Ph.D. Population genetics of polar bears (Ursus maritimus) in the Svalbard area. Collaboration: University of Oslo (Natural History Museum) and Norwegian Polar Institute, Tromsø.</p>
                    <ul>
                        <li>Ph.D trial lecture: Microevolutionary responses to climate change and harvesting in wildlife: evidence and implications for management.</li>
                    </ul>
                </section>

                <section id="master2">
                    <p class="greyband">2002-2003 Second year Master-degree. Collaboration University Claude Bernard (UCB), Lyon I and Universitetet Joseph Fourier (UJF), Grenoble, France.</P>
                    <p>Courses (UCB): Analysis and modelling of biological systems (Statistics and modelling applied to biological research).</p>
                    <p>Research (UJF): Development and testing molecular methods to fight wildlife criminality (see publications).</p>

                </section>

                <section id="master1">
                    <p class="greyband">2001-2002 First year Master-degree. Population genetics, population biology and ecosystems. UJF, France.</p>
                    <p>Including research project training (see publications).</p>
                </section>

                <section id="bachelor3">
                    <p class="greyband">2000-2001 Third year bachelor-degree: UJF, France.</p>
                    <p>Additional during summer vaccations: 2 months research project (see publications).</p>
                </section>

                <section id="bachelor2">
                    <p class="greyband">1999-2000 Second Year bachelor-degree: University of Burgundy, Dijon, France. Cellular biology, microbiology, immunologi.</p>
                    <p> Additional special courses: bioinformatics, marketing.</p>
                </section>

            </section>

            <section class = "innerbox">
                <h2 class="diverse0">Diverse</h2>
                <section id="Languages" class="diverse1">
                    <p class="greyband">Languages</p>
                    <ul>
                        <li>French: Mother tongue.</li>
                        <li>English: Fluent.</li>
                        <li>Norwegian: Orally fluent, written: good. (Bergenstest: B2).</li>
                        <li>German: (some understanding).</li>
                    </ul>
                </section>

                <section id="Computer" class="diverse2">
                    <p class="greyband">Computer literacy</p>
                    <p>OS: Ubuntu - Windows</p>
                    <p>Programming languages: R, bash, python, SQLite, nextflow, (HTML and CSS: beginner).</p>
                    <p>Some experience with making/using containers (Docker, singularity).
                    <p>Note: Most of my Github repositories (work at the Veterinary Institute) are private.</p>
                </section>

                <section id="Voluntary" class="diverse3">
                    <p class="greyband">Voluntary:</p>
                    <p>Coinstructor and helper at <a href="https://www.uio.no/english/for-employees/support/research/research-data/training/carpentry/">Carpentries Oslo</a> and part of the <a href="https://uio-carpentry.github.io/studyGroup/">Oslo Carpentry study group</a>.<br>Aim: become a certified Carpentry instructor (checkout in progres).</p>
                </section>

            </section>

            <section class="results">
                <h2>Results and publications</h2>
                <section id="fromBioinfo">
                    <p class="greyband"> <strong>Accepted: 2019.</strong> Resulting from bioinformatics reconversion trainee.</p>
                    <p><em>contributions: Formal analysis, Investigation, Methodology, Software, Writing, Review & Editing.</em></p>
                    <p>Håkon Kaspersen, Eve Zeyl Fiskebeck, Camilla Sekse, Jannice Schau Slettemeås, Anne Margrete Urdahl, Madelaine Norström, Karin Lagesen, Roger Simm. 2019. Comparative genome analyses of wild type- and quinolone resistant <em>Escherichia coli</em> indicate dissemination of QREC in the Norwegian broiler breeding pyramid. (In press: <a href="https://www.frontiersin.org/">Frontiers</a>).</p>
                    <p>Håkon Kaspersen, Camilla Sekse, Eve Zeyl Fiskebeck, Jannice Schau Slettemeås, Roger Simm, Madelaine Norström, Anne Margrete Urdahl, Karin Lagesen. 2019. Dissemination of quinolone resistant <em>Escherichia coli</em> in the Norwegian broiler and pig production chain, and possible persistence in the broiler production environment. (Accepted: <a href="https://aem.msubmit.net/cgi-bin/main.plex?el=A4GH6BZlU7A5EERE7F3A9ftd3HsLk6rOBhlG6VJKvJ6YTAZ">Applied and Environmental Microbiology</a>).</p>
                </section>

                <section id="published">
                    <p class="greyband"><strong>Published</strong></p>
                    <section id="from-lab">
                        <p class="greyband">Resulting from laboratory engineer work.</p>
                        <p>Zeyl, E.; Marcotegui, P.; Harris, P.D.; Bakke, T.A.; Bachmann, L. (2012). New parasites from old fishes. <a href="https://drive.google.com/file/d/114pLtvf4t5OkmzjnQasriqZ8R_B3Lakf/view?usp=sharing">Naturen, 6, 27-280.</a>(In Norwegian).</p>
                    </section>
                    <section id="from-phd">
                        <p class="greyband">Resulting from Ph.D. work.</p>
                        <p>Peacock, Elisabeth et al. (2015). Implications of the circumpolar genetic structure of polar bears for their conservation in a rapidly warming Arctic. <a href="https://drive.google.com/file/d/1620LtGYMGVwwhKh0xiWuxAR0i-FWnbhO/view?usp=sharing">PLoS ONE,Volum 10:0112021.(1) s.</a></p>
                        <p>Zeyl E. Population genetics of polar bears in the Svalbard area. (2010). University of Oslo, Natural History Museum (<a href="https://drive.google.com/file/d/1gHf9MGTctjMo7AhLA1zqHkOba9Ws3MP7/view?usp=sharing">Ph.D. thesis</a>. With additional data).</p>
                        <p>Zeyl E., Ehrich D., Aars J., Bachmann L., Wiig Ø. (2010). Denning area fidelity and mitochondrial DNA diversity of female polar bears (<em>Ursus maritimus</em>) in the Barents Sea area. <a href="https://drive.google.com/file/d/1-jz_0qygQdLQqSTlKhTDeaIJEAF4feAL/view?usp=sharing">Canadian Journal of Zoology, 88, 1139-1148.</a></p>
                        <p>Lindqvist C., et al. (2010). Complete mitochondrial genome of a Pleistocene jawbone unveils the origin of polar bear. <a href="https://drive.google.com/file/d/1wKoPd56D7IlyO3VSx6_x6gxri09Da01v/view?usp=sharing">Proceedings of the National Academy of Sciences, 106, 5053-5057.</a></p>
                        <p>Zeyl E., Aars J., Ehrich D., Bachmann L., Wiig Ø. (2009). The mating system of polar bears: a genetic approach. <a href="https://drive.google.com/file/d/1-75EMWpR7SJTQzatddRdZwkei8MqfSEu/view?usp=sharing">Canadian Journal of Zoology, 87, 1195-1209.</a></p>
                        <p>Zeyl E., Aars J., Ehrich D., Wiig Ø. (2009). Families in space: relatedness in the Barents Sea population of polar bears (<em>Ursus maritimus</em>). <a href="https://drive.google.com/file/d/1_nTUFHxKjSO0XPU7Phk99v6CORfkWVXK/view?usp=sharing">Molecular Ecology,18, 735-749.</a></p>
                    </section>

                    <section id="fromMaster2">
                        <p class="greyband">Resulting from Master work (second year)</p>
                        <p>Beja-Pereira A., Zeyl E., Ouragh L., Nagash H., Ferrand N., Taberlet P., Luikart G. (2004). Twenty polymorphic microsatellites in two of North African most threatened ungulates: <em>Gazella dorcas</em> and <em>Ammotragus lervia</em> (Bovidae, Artiodactyla). <a href="https://drive.google.com/file/d/1SrysrUWA6rHnSBXFjcaYEh2QEu_A2zo4/view?usp=sharing">Molecular Ecology Notes, 4, 452-455.</a></p>
                        <p>Maudet C., Beja-Pereira, A., Zeyl E., Nagash, H., Kence A., Özüt D., Biju-Duval, M.-P., Boolormaa S., Coltman D.W., Taberlet P., Luikart G. (2004). A standard set of polymorphic microsatellites for threatened mountain ungulates (Caprini, Artiodactyla). <a href="https://drive.google.com/file/d/1QPvzVHGBf_NC46O1pgUbE-5jboXhy4PN/view?usp=sharing">Molecular Ecology Notes, 4, 49–55.</a></p>
                        <p>Zeyl E. (2003). Wildlife forensics: developing and evaluating molecular tools to assign Middle East wild sheep (<em>Ovis spp.</em>) to their population of origin. Usefulness to fight poaching (<a href="https://drive.google.com/file/d/1jdrBAe83KkBKe9vdUZdtSm6PfI81jB_i/view?usp=sharing">Master degree research thesis, in French).</a></p>
                        <p>Zeyl E. (2003). Uses of molecular genetics tools in wildlife forensics (<a href="https://drive.google.com/file/d/1oTpGKGxIS3viexvGwHv-mVyG8Vs8mfxl/view?usp=sharing">Master degree bibliographic thesis, in French).</a></p>
                    </section>

                    <section id="fromMaster1">
                        <p class="greyband">Resulting from Master work (first year)</p>
                        <p>Zeyl E. (2003). Birth phenology in a chamois (<em>Rupicapra rupicapra</em>) population (Bauges, France).  <a href="https://drive.google.com/file/d/14gOZhTBFTt1-4z4LMAm610bKcXdgGcr0/view?usp=sharing">Data analysis: from years 1993 to 2002</a>. (Research Report In French).</p>
                    </section>

                    <section id="frombachelor3">
                        <p class="greyband">Resulting from bachelor work (third year)</p>
                        <p>Zeyl E. and De Mita S. (2001). Locomotory behavioral study of a grasshopper species (<em>Podisma pedestris</em>) living in the upper Alps. Developing and testing several techniques to mark-recapture grasshoppers. (Research report: In French).</p>
                    </section>

                </section>

                <section id="">
                </section>


            </section>

            <section class="foot">
                <p></p>
            </section>
        </div>

    </main>
</body>




<footer>
  <p></p>
</footer>

</html>
